src/db-utils.js

```diff
@@ -64,4 +66,9 @@ export async function weightStats(username) {
             res(data);
         });
     });
+}
+
+export function dumpDB() {
+    const data = fs.readFileSync(DB_LOCATION, {encoding:'utf8', flag:'r'});
```

Comment:
We don't need to block the PR on this but we need to enforce a maximum for the file size we are willing to dump.